src/middleware/navigation/update/has.presenter.middleware.ts

```diff
@@ -0,0 +1,8 @@
+import { Request, Response, NextFunction } from 'express';
+
+import { checkIfPresenterDataExists } from '../common-code/has.presenter';
+import { UPDATE_LANDING_PAGE_URL } from '../../../config';
+
+export const hasUpdatePresenter = (req: Request, res: Response, next: NextFunction): void => {
+  checkIfPresenterDataExists(req, res, next, UPDATE_LANDING_PAGE_URL);
```

Comment:
I see the registration has SOLD_LAND_FILTER for 3rd arg but we have the landing page. Think we should also pass out first page (secure-update-filter).

Reply:
fixed, changed both to redirect to SECURE_UPDATE_FILTER_URL